frontend/src/app/commonComponents/Checkboxes.tsx

```diff
@@ -79,6 +81,7 @@ const Checkboxes = (props: Props) => {
               type="checkbox"
               value={value}
               name={name}
+              ref={inputRef}
```

Comment:
I've never seen a group of checkboxes share the same `ref`! Which one of them is set in `.current`?